src/mds/nameserver2/file_lock.cpp

```diff
@@ -117,10 +117,10 @@ void FileLockManager::LockInternal(const std::string& path,
 
     if (lockType == kRead) {
         // get read lock
-        entry->rwLock_.RDLock();
+        bthread_rwlock_rdlock(&entry->rwLock_);
```

Comment:
why chanage this lock to bthread lock

Reply:
> why chanage this lock to bthread lock

The original way will have some problems in the higher version of gcc